app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CustomerItemsController < ApplicationController
+    def create
+      customer = Customer.find(params[:id])
+      item = Item.find(params[:item_id])
+      customer.items << item
```

Comment:
I'm ok with this here, however I think it would be even better if you had made a method in the Customer model called `add_item(item)` that had this line in it, then line 5 here would call `customer.add_item(item)`. It's a small change, but then it allows for future changes on how an item is added to happen within the model method.